libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/arrow/api/MapboxRouteArrowView.kt

```diff
@@ -196,4 +232,9 @@ class MapboxRouteArrowView(private val options: RouteArrowOptions) {
             style.getSourceAs<GeoJsonSource>(sourceId)?.featureCollection(featureCollection)
         }
     }
+
+    private fun rebuildLayersAndSources(style: Style, options: RouteArrowOptions) {
+        RouteArrowUtils.removeLayers(style)
+        initializeLayers(style, options)
+    }
```

Comment:
Do we need this function? Can't we run `RouteArrowUtils.removeLayers(style)` always from inside of `initializeLayers(style, options)`?

Reply:
Also, we should rename `RouteArrowUtils.removeLayers` to `RouteArrowUtils.removeLayersAndSources`.